aptos-move/framework/aptos-stdlib/sources/cryptography/multi_ed25519.move

```diff
@@ -77,6 +83,34 @@ module aptos_std::multi_ed25519 {
     // Functions
     //
 
+    #[test_only]
+    public fun generate_keys(threshold: u8, n: u8): (SecretKey,ValidatedPublicKey) {
+        assert!(1 <= threshold && threshold <= n, error::invalid_argument(E_INVALID_THRESHOLD_OR_NUMBER_OF_SIGNERS));
+        let (sk_bytes, pk_bytes) = generate_keys_internal(threshold, n);
+        let sk = SecretKey {
+            bytes: sk_bytes
+        };
+        let pk = ValidatedPublicKey {
+            bytes: pk_bytes
+        };
+        (sk, pk)
+    }
+
+    #[test_only]
+    public fun multi_sign_arbitrary_bytes(sk: &SecretKey, msg: vector<u8>) : Signature {
```

Comment:
Actually, let's call this `sign_arbitrary_bytes` to be consistent with `ed25519`, as we already are with the other functions.